packages/react-instantsearch/examples/autocomplete/README.md

```diff
@@ -0,0 +1,8 @@
+This example shows how to build an autocomplete with react-instantsearch
+
+To start the example: 
+
+* npm install
+* npm start
+
+Read more about react-instantsearch [in our documentation](https://community.algolia.com/instantsearch.js/react/).
```

Comment:
I'm not sure, at least, not with this solution. 
What I can explain on a guide is the idea behind the example: having a function listening to all changes and then construct the autocomplete state with it. But the way we can achieve this now seems very fragile, that's why I'm not sure about a dedicated guide in the end.

But I can still do that in the README, and then if we come up with a good solution to simplify this kind of use-cases, then we could write a guide. We also need to add all the recipes to the documentation website.